CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_greenhouse v0.8.0
+[PR #28](https://github.com/fivetran/dbt_greenhouse/pull/28) is a breaking change due to [upstream updates](
+https://github.com/fivetran/dbt_greenhouse_source/releases/tag/v0.8.0):
+
+## Breaking Changes
+- Updated `*_id` fields in upstream `stg_*` models to be cast as strings to ensure compatibility in downstream joins by avoiding potential type mismatches.
+  - Note: most IDs were previously stored as integers, so `*_id` fields in the end models will now also be strings.
+
+## Documentation
+- Update dbt documentation definitions.
```

Comment:
Is this referring to the yml updates? Looks like fields were added but no definitions were updated.

Reply:
`greenhouse.yml` was updated. Or is the issue that you want it to be "added" instead of "updated".